static/src/js/containers/MapContainer/MapContainer.js

```diff
@@ -75,94 +99,74 @@ const EPSG3031 = new window.L.Proj.CRS(
 )
 const crsProjections = [EPSG3413, EPSG4326, EPSG3031]
 
-const urlPropsQueryConfig = {
-  map: {
-    type: UrlQueryParamTypes.string,
-    queryParam: 'm'
-  }
-}
-
 const mapDispatchToProps = dispatch => ({
-  onChangeMap: ({ map }) => {
-    dispatch(actions.changeMap(map))
-  }
+  onChangeMap: query => dispatch(actions.changeMap(query))
 })
 
-// const mapStateToProps = state => ({
-//   map: state.map
-// })
+const mapStateToProps = state => ({
+  mapParam: state.map.mapParam
+})
 
-class EdscMap extends Component {
+export class EdscMapContainer extends Component {
   constructor(props) {
     super(props)
-    this.handleMoveend = this.handleMoveend.bind(this)
-  }
 
-  componentDidMount() {
-    const { map, onChangeMap } = this.props
-    onChangeMap({ map })
-  }
-
-  componentWillReceiveProps(nextProps) {
-    const { map, onChangeMap } = this.props
-    if (map !== nextProps.map) {
-      onChangeMap({ map })
-    }
+    this.handleMoveend = this.handleMoveend.bind(this)
   }
 
   componentDidUpdate() {
+    const {
+      leafletElement: map = null
+    } = this.mapRef
+
     if (this.mapRef) {
-      this.mapRef.leafletElement.invalidateSize()
+      map.invalidateSize()
     }
   }
 
   handleMoveend(event) {
     const map = event.target
-    console.log('map', map)
+    // console.log('map', map)
     const center = map.getCenter()
     const { lat } = center
     const { lng } = center
-    console.log('lat, lng', lat, lng)
+    // console.log('lat, lng', lat, lng)
```

Comment:
Another debug? If so we should implement a flag or something for this type of stuff.